wiki/osu!_File_Formats/Osu_(file_format)/en.md

```diff
@@ -329,7 +329,7 @@ The third object is a slider:
 - At the position (100,100)
 - At 12.6 seconds
 - Starting a new combo
-- With a compound bézier curve slider body, where the first curve's control points are (100,100), (200,200), and (250,200), and the second curve's control points are (250,200), (300,150)
```

Comment:
This wording demonstrates that these are indeed two separate [bézier curves](https://en.m.wikipedia.org/wiki/B%C3%A9zier_curve) joined together, forming a [composite bézier curve](https://en.m.wikipedia.org/wiki/Composite_B%C3%A9zier_curve) as stated, with the duplicate coordinates indicating the separation. I don't see a reason for omitting this.

Reply:
but two separate bézier curves isn't mean two slider control point, actually this slider have 4 point.
this is my opinion at the time
so I delete first control point and second control point

maybe use `The first Bezier curve` may better?
![image](https://user-images.githubusercontent.com/34775378/124132432-c7c13680-dabb-11eb-9464-be11c64b764c.png)
this is the slider actually look like